archives/2020/student-reports/improve-and-maintain-25-activities-Jui-Pradhan.md

```diff
@@ -2,55 +2,67 @@ Improve and maintain atleast 25 Sugar Activities
 --------------------------
 ***Google Summer of Code, 2020***
 
-My work includes feature additions, bug fixes, adding support for different display resolutions, few design changes, port to python3 as a part of improving Sugar Activities. This project also aimed at maintaining activities, the work for which included: making activity releases, adding or fetching translations, flake8 changes and testing activities extensively. Moreover, fixing issues, reviewing pull requests and finishing unmerged work was also a part of maintenance.
+*Name - Jui Pradhan*
```

Comment:
You have modified @JuiP's work product. Was this intended?

Reply:
No Actually I just rebase my master branch with current master.

Reply:
@pro-panda Removed those commits. @quozl @chimosky Please review and merge as I need link to submit in evaluation.